JS-Algorithms/challenges.js

```diff
@@ -96,6 +139,18 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 
 const digitSum = (n) => {
   // YOUR CODE HERE...
+
+  const GetFactorial = ({ n }) => {
+    return Array.from({ length: n }, (_, i) => BigInt(i + 1)).reduce(
+      (acc, curr) => acc * curr,
+      1n
+    );
+  };
+
+  const factorial = GetFactorial({ n });
+  const NumArray = `${factorial}`.split("");
+
+  return NumArray.reduce((acc, element) => parseInt(acc) + parseInt(element));
```

Comment:
We can avoid using string functions on arhitmetical purposes and make the calculation using a for loop this could help to avoid parsing the result